src/main/java/org/jenkinsci/remoting/engine/JnlpProtocol.java

```diff
@@ -58,6 +58,14 @@
     }
 
     /**
+     * Whether this protocol is enabled for connecting.
+     * @return {@code true} if this protocol is enabled.
+     */
+    public boolean isEnabled() {
+        return !Boolean.getBoolean(getClass().getName()+".disabled");
```

Comment:
so we retrieve the property every time. Useful for debugging, performance should not suffer in this place